packages/embark-swarm/src/index.js

```diff
@@ -96,8 +91,27 @@ class Swarm {
 
       upload_swarm.deploy(readyCb);
     });
+  }
+
+  setupSwarmAPI() {
+    this.events.request("runcode:whitelist", 'swarm-api', () => {});
+    this.events.on("storage:started", async () => {
+      await this.events.request2("runcode:register", "swarm", this.swarm);
+      await this.events.request2('console:register:helpCmd', {
+        cmdName: "swarm",
+        cmdHelp: __("instantiated swarm-api object configured to the current environment (available if swarm is enabled)")
+      });
+    });
+  }
+
+  setupEmbarkJS() {
+    this.events.request("embarkjs:plugin:register", 'storage', 'swarm', 'embarkjs-swarm');
+    this.events.request("embarkjs:console:register", 'storage', 'swarm', 'embarkjs-swarm');
```

Comment:
Shouldn't this be an async method with:
```suggestion
    await this.events.request2("embarkjs:console:register", 'storage', 'swarm', 'embarkjs-swarm');
```
?